io-backend-projects/core/05_tlscert-prod-practices-ipatente-io-pagopa-it.tf

```
@@ -0,0 +1,92 @@
+    credential_key_vault_resource_group = local.prod_key_vault_resource_group
+    service_connection_ids_authorization = [
+      module.PROD-TLS-AZDO-CERT-SERVICE-CONN.service_endpoint_id,
+    ]
+  }
+  tlscert-prod-practices-ipatente-io-pagopa-it-variables = {
+    KEY_VAULT_SERVICE_CONNECTION = module.PROD-TLS-AZDO-CERT-SERVICE-CONN.service_endpoint_name,
+    KEY_VAULT_NAME               = local.prod_key_vault_name
+  }
+  tlscert-prod-practices-ipatente-io-pagopa-it-variables_secret = {
+  }
+}
+
+# change only providers
+#tfsec:ignore:general-secrets-no-plaintext-exposure
+module "tlscert-prod-practices-ipatente-io-pagopa-it-cert_az" {
+  source = "github.com/pagopa/azuredevops-tf-modules//azuredevops_build_definition_tls_cert_federated?ref=v7.2.0"
+  count  = var.tlscert-prod-practices-ipatente-io-pagopa-it.pipeline.enable_tls_cert == true ? 1 : 0
+
+  project_id                   = azuredevops_project.project.id
+  repository                   = var.tlscert-prod-practices-ipatente-io-pagopa-it.repository
+  path                         = "${local.domain}\\${var.tlscert-prod-practices-ipatente-io-pagopa-it.pipeline.path}"
+  github_service_connection_id = azuredevops_serviceendpoint_github.azure-devops-github-ro.id
+
+  dns_record_name                      = var.tlscert-prod-practices-ipatente-io-pagopa-it.pipeline.dns_record_name
+  dns_zone_name                        = var.tlscert-prod-practices-ipatente-io-pagopa-it.pipeline.dns_zone_name
+  dns_zone_resource_group              = local.tlscert-prod-practices-ipatente-io-pagopa-it.dns_zone_resource_group
+  tenant_id                            = local.tlscert-prod-practices-ipatente-io-pagopa-it.tenant_id
+  subscription_name                    = local.tlscert-prod-practices-ipatente-io-pagopa-it.subscription_name
+  subscription_id                      = local.tlscert-prod-practices-ipatente-io-pagopa-it.subscription_id
+  managed_identity_resource_group_name = local.identity_rg_name
+
+  location                            = local.location
+  credential_key_vault_name           = local.tlscert-prod-practices-ipatente-io-pagopa-it.credential_key_vault_name
+  credential_key_vault_resource_group = local.tlscert-prod-practices-ipatente-io-pagopa-it.credential_key_vault_resource_group
+
+  variables = merge(
+    var.tlscert-prod-practices-ipatente-io-pagopa-it.pipeline.variables,
+    local.tlscert-prod-practices-ipatente-io-pagopa-it-variables,
+  )
+
+  variables_secret = merge(
+    var.tlscert-prod-practices-ipatente-io-pagopa-it.pipeline.variables_secret,
+    local.tlscert-prod-practices-ipatente-io-pagopa-it-variables_secret,
+  )
+
+  service_connection_ids_authorization = local.tlscert-prod-practices-ipatente-io-pagopa-it.service_connection_ids_authorization
+
+  schedules = {
+    days_to_build              = ["Mon", "Wed", "Fri"]
+    schedule_only_with_changes = false
+    start_hours                = 4
+    start_minutes              = 30
```

```suggestion
    start_minutes              = 35
```
I suggest to move trigger a little bit forward to not overlap with other pipelines